dev/traffic_router/conf/trafficrouter.dev.ciab.test.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I don't know if there's a standard "comment" grammar structure in certificate syntax - that should work with implementations using openssl, but I believe Macs don't use that SSL library because it's "too open-source" or somesuch, and I'm not sure that all SSL implementations allow for arbitrary text outside the `BEGIN` and `END` statements. Do they?